tests/unit/test_guid_validation.py

```diff
@@ -0,0 +1,9 @@
+from django_guid.middleware import GuidMiddleware
+
+
+def test_valid_guid():
+    assert (GuidMiddleware._validate_guid("07742cab407e4e8089ebfd191acbb752") is True)
```

Comment:
Double quotes :eyes:

Reply:
Fixed. This is what you get when you don't lint your tests 🤷